packages/components/src/components/RunHeader/RunHeader.jsx

```diff
@@ -23,19 +23,43 @@ export default function RunHeader({
   description,
   displayRunHeader = true,
   duration = null,
-  resource,
   lastTransitionTime,
   loading,
   message,
   namespace,
+  pipelineRunError,
   reason,
+  resource,
   runName,
+  showFailureMessage,
   status,
   triggerHeader,
   triggerInfo
 }) {
   const intl = useIntl();
 
+  const errorNotification = (pipelineRunError || showFailureMessage) && (
+    <InlineNotification
+      kind="error"
+      hideCloseButton
+      lowContrast
+      title={intl.formatMessage(
+        {
+          id: pipelineRunError
+            ? 'dashboard.pipelineRun.failedMessage'
+            : 'dashboard.pipelineRun.ErrorMessage',
+          defaultMessage: pipelineRunError
+            ? 'Unable to load PipelineRun: {reason}'
+            : 'PipelineRun Error: {reason}'
+        },
```

Comment:
This will need to be handled as 2 separate instances of `intl.formatMessage`, i.e.

`pipelineRunError ? intl.formatMessage({ id: '….failedMessage', … }) : intl.formatMessage({ id: '….errorMessage', … })`

Also the last segment of the key should start with lowercase:
```diff
-dashboard.pipelineRun.ErrorMessage
+dashboard.pipelineRun.errorMessage
```

Reply:
I didn't see this comment before my last update - changing `Error` > `error` now